### DISCUSSION TITLE
Facing issue in marker not showing in circle

### DISCUSSION THREAD

**Discussion by pankajsoftyoi** [react-native-maps/react-native-maps#4376] [Q&A]
### Summary

I have added circle in map, but some how in few cases marker not showing in circle,

<img width="280" alt="Screenshot 2022-08-03 at 5 37 50 PM" src="https://user-images.githubusercontent.com/106329563/182604372-8061b17c-c781-4e53-8cf0-4a6e76119be3.png">
<img width="269" alt="Screenshot 2022-08-03 at 5 37 36 PM" src="https://user-images.githubusercontent.com/106329563/182604378-f0bcdf2f-7a88-499f-80f3-9c8eadf1092b.png">
<img width="287" alt="Screenshot 2022-08-03 at 5 38 04 PM" src="https://user-images.githubusercontent.com/106329563/182604348-d5b805de-247c-46d5-b4c8-eabee8875693.png">

Please let me know if anyone have solution for this issue.

### Reproducible sample code

```TSX
{(showRadius || showTotalRadius) && latitude != null && longitude != null && (
              <Circle
                center={{ latitude: +circleCenter.latitude, longitude: +circleCenter.longitude }}
                radius={distanceCircle > 0 ? distanceCircle : 0}
                fillColor='rgba(1, 128, 128, 0.2)'
                strokeColor={colors.secondary1}
                zIndex={2}
                strokeWidth={3}
              />
            )}
            {!!entries &&
              entries.length > 0 &&
              !showRadius &&
              entries.map((property: PropertyType, index) => (
                <PriceMarker
                  zIndex={index + 1}
                  key={property.propertyId}
                  latitude={property.latitude}
                  longitude={property.longitude}
                  property={property}
                  onMarkerPress={onMarkerPress}
                  selected={property.propertyId === mapSelectedProperty?.propertyId}
                  amount={property.effMinPrice}
                />
              ))}
```

### Steps to reproduce

Circle add in map

### Expected result

marker display in circle

### Actual result

marker not showing in circle

### React Native Maps Version

0.27.1

### What platforms are you seeing the problem on?

iOS (Google Maps)

### React Native Version

0.63.4

### What version of Expo are you using?

Not using Expo

### Device(s)

iPhone 11

**Comment by FrSenpai** [comment]
Same here, marker is displayed, but take the opacity of the circle (zindex problem probably ?)

It's hapening only when I use custom marker with an image on it
